src/core/tests/pass/serialization/deterministicity.cpp

```diff
@@ -193,6 +193,12 @@ TEST_P(SerializationDeterministicityInputOutputTest, FromOvModel) {
     auto& expected1 = modelRef;
     ov::pass::Serialize(m_out_xml_path_1, m_out_bin_path_1, irVersion).run_on_model(modelRef);
     auto expected2 = ov::test::readModel(m_out_xml_path_1, m_out_bin_path_1);
+
+    // We need to check and erase this entry because it's expected to be different in these two scenarios.
+    std::string path_key = "weights_path";
```

Comment:
The problem is that the function reading the model is plugin-agnostic - it will set the weights_path regardless of the plugin(s) which will be used with the model later. Do you suggest that the function reading the model should set the weights_path conditionally?

Reply:
Removed - it's no longer serialized.